src/Test/Utilities/Desktop/TestResource.resx

```diff
@@ -123,7 +123,7 @@
 #pragma warning disable 414, 3021
```

Comment:
Actually they get displayed in the error list, but only when you have the file open (seems like a bug, the IDE should know it's a resource). They are not project build errors though, so I think that's fine. We could change the extension if we really wanted to avoid these.